src/services/install/default-arguments.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import { Logger } from '../../logger'
+import fs from 'fs'
+
+const logger = Logger('install-default-arguments')
+
+export async function defaultArguments() {
+
+    const gitHubResponse = (await axios.get(
+        'https://hub.docker.com/v2/repositories/backendless/bl-server/tags/?page_size=25&page=1&ordering=last_updated'))
+        .data
+
+    logger.verbose(`response from github: ${JSON.stringify(gitHubResponse)}`)
+    const version = gitHubResponse.results
+        .map(result => result.name)
+        //get first version which ands with number. Because we should skip dev, stage and so on version
+        .filter(name => /[0-9]+$/.test(name))[0]
+
+    const mountPath = `${process.cwd()}/k8s-mount`
+
+    if (!fs.existsSync(mountPath)) {
```

Comment:
`Sync` methods are out of favor =) because it blocks the event loop

there is a good module with promisified FS methods 

https://www.npmjs.com/package/fs-extra